### ISSUE TITLE
Have you trained in celebA dataset?

### ISSUE THREAD

**Issue by zhangqianhui** [martinarjovsky/WassersteinGAN#28]
In my experiments , Wgan in celeba generate more low quality images after 100000 iters compared to DCGAN.
I don't know why.

**Comment by LukasMosser** [comment]
Care to post your discriminator loss vs generations?
And your hyperparameters? Any other changes, maybe then someone can help.
You'll need to provide a bit more info.

**Comment by zhangqianhui** [comment]
@LukasMosser  And the all parameters is the default.And the celeba dataset is center-cropped into 108x108 pixels ,then resize 64x64 as the input of generator.

**Comment by martinarjovsky** [comment]
I trained on celebA (in fact, this was the dataset I used before moving into bedrooms because faces are apparently too easy a dataset). Everything worked fine, I used the dataloader from Soumith's original DCGAN code, see https://github.com/soumith/dcgan.torch and specifically https://github.com/soumith/dcgan.torch/blob/master/data/crop_celebA.lua .

Best!
Martin

**Comment by yichuan9527** [comment]
In my experiment, using the same learning rate, WGAN convergence faster than DCGAN. However, after the iteration of 100000 steps,  get some good results. After the iteration of 150,000 steps, the generator began to fool the discriminator.
